client/my-sites/stats/stats-module/connected-list.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import ErrorPanel from '../stats-error';
+import StatsList from '../stats-list';
+import StatsListLegend from '../stats-list/legend';
+import DatePicker from '../stats-date-picker';
+import Card from 'components/card';
+import StatsModulePlaceholder from './placeholder';
+import SectionHeader from 'components/section-header';
+import QuerySiteStats from 'components/data/query-site-stats';
+import { getSelectedSite } from 'state/ui/selectors';
+import {
+	isRequestingSiteStatsForQuery,
+	getSiteStatsParsedData
+} from 'state/stats/lists/selectors';
+
+const StatsConnectedModule = React.createClass( {
```

Comment:
I'd suggest moving away from `React.createClass` . Some debate the merits of `class` generally, but fact remains that the `React.createClass` method is effectively deprecated ([source](https://github.com/reactjs/core-notes/blob/19b13888c8ef9890f96301d785aca78d69d90efa/2016-05/may-19.md#createclass-vs-es2015-classes)).